Sources/SwiftCliCore/Chartify.swift

```diff
@@ -6,17 +6,19 @@ public final class Chartify {
     var inputValidator: InputValidator
     var chartConstructor: ChartConstructor
     var fileValidator: FileValidator
+    var writeToFile: WriteToFile
 
 
-    public init(inputValidator: InputValidator, chartConstructor: ChartConstructor, fileValidator: FileValidator) {
+    public init(inputValidator: InputValidator, chartConstructor: ChartConstructor, fileValidator: FileValidator, writeToFile: WriteToFile) {
 
         self.inputValidator = inputValidator
         self.chartConstructor = chartConstructor
         self.fileValidator = fileValidator
+        self.writeToFile = writeToFile
     }
 
 
-    public func run(userInput: [String], file: String? = nil, knitFlat: Bool = false) {
+    public func run(userInput: [String], file: String? = nil, knitFlat: Bool = false, fileNameToWrite: String? = nil) {
```

Comment:
Also nonblocking for this PR but: does `Chartify` need to know or care about what output file name to write to? Or is that something that it makes more sense for `FileWriter` to know?